javascript-data-model/order-history.js

```diff
@@ -0,0 +1,30 @@
+const orderHistory = [
+  {
+    order: '114-3941689-8772232',
+    title: 'Javascript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    price: 31.55,
+  },
+  {
+    order: '113-9984268-1280257',
+    title: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    price: 41.33,
+  },
+  {
+    order: '113-2883177-2648248',
+    product:
+      'GameCube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+    company: 'Nintendo',
+    price: 15.98,
+  },
+  {
+    order: '113-28831777-2648248',
+    product1: 'GameCube Controller-Super Smash Bros. Edition(Nintendo Switch)',
+    company: 'Nintendo',
+    price: 94.95,
+    product2: 'The Art of Sql',
```

Comment:
This is a good attempt! However it may be hard to scale this up. What if an order has 10 items? The way it's written you'd have to manually add properties for `product3` `product4` and so on.

This would be a perfect plays to create a new property called `products` that is an array where each index is a `product` that is itself an object with the relevant properties.

I'll demonstrate what I'm talking about in class!